include/libdnf/rpm/package.hpp

```diff
@@ -0,0 +1,287 @@
+    ReldepList get_enhances() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.supplements
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_supplements(DnfPackage * pkg)
+    ReldepList get_supplements() const;
+
+    // REPODATA
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.baseurl
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_baseurl(DnfPackage * pkg)
+    std::string get_baseurl();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.location
+    /// @replaces dnf:dnf/package.py:attribute:Package.relativepath
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_location(DnfPackage * pkg)
+    std::string get_location();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.chksum
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_chksum(DnfPackage *pkg, int *type)
+    Chksum get_chksum();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.hdr_chksum
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_hdr_chksum(DnfPackage *pkg, int *type)
+    Chksum get_hdr_chksum();
+
+    /// TODO get_changelogs - requires changelog
+    /// @replaces dnf:dnf/package.py:attribute:Package.changelogs
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_changelogs(DnfPackage * pkg)
+    // void get_changelogs() const;
+
+    // SYSTEM
+
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_installed(DnfPackage * pkg)
+    /// @replaces dnf:dnf/package.py:attribute:Package.installed
+    bool is_installed() const;
+
+    /// TODO is_local
+    /// @replaces dnf:dnf/package.py:method:Package.localPkg(self)
+    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
+    //bool is_local() const;
+
+    /// TODO get_from_repo_id - requires swdb
+    /// For an installed package, return repoid of repo from the package was installed.
+    /// For an available package, return an empty string.
+    ///
+    /// @replaces dnf:dnf/package.py:attribute:Package.ui_from_repo
+    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_origin(DnfPackage * pkg)
+    // void get_from_repo_id() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.hdr_end
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_hdr_end(DnfPackage * pkg)
+    unsigned long long  get_hdr_end() noexcept;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.installtime
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
+    unsigned long long  get_install_time() noexcept;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.medianr
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_medianr(DnfPackage * pkg)
+    unsigned long long  get_medianr() noexcept;
```

Comment:
Name of this attribute is not quite obvious.
I'd consider renaming it to `get_media_number` or at least add a comment explaining what it contains.

Reply:
Renamed